### ISSUE TITLE
Settings editor high contrast dark theme doesn't have good listview focus highlights

### ISSUE THREAD

**Issue by rzhao271** [microsoft/vscode#165530]
1. Open the Settings editor.
2. Use the dark high contrast theme.
3. Search for the `files.exclude` setting.
4. Tab until keyboard focus goes to the listview.
5. Press the down and up keys. The pencil and close icons on the right side should move.
6. :bug: There's no background colour change accompanying the keyboard focus.

This problem does not occur with the high contrast light theme.
